k8s/service_controller/tests/sb_create_test.go

```diff
@@ -0,0 +1,45 @@
+package tests
+
+import (
+	"testing"
+
+	"bytes"
+	"net/http"
+	"os/exec"
+)
+
+func Test_SB_create_basic(t *testing.T) {
```

Comment:
in fact, you can get rid of this whole test file if this doesn't get uncommented.